docs/migrations/v7_1-to-v7_3.md

```diff
@@ -0,0 +1,41 @@
+# Migrating from v7.1 to v7.3
+
+This guide provides instructions for migrating to version `v7.3.0` of ibc-go.
+
+There are four sections based on the four potential user groups of this document:
+
+- [Migrating from v7.1 to v7.3](#migrating-from-v71-to-v73)
```

Comment:
Is it supposed to be v7.1 -> v7.3? Why not v7.2?

```suggestion
- [Migrating from v7.2 to v7.3](#migrating-from-v71-to-v73)
```

Reply:
No particular reason, I am fine changing to v7.2.